OpenUtau.Core/Classic/VoicebankErrorChecker.cs

```diff
@@ -108,6 +117,29 @@ public class VoicebankErrorChecker {
             try {
                 using (var wav = Core.Format.Wave.OpenFile(filePath)) {
                     fileDuration = wav.TotalTime.TotalMilliseconds;
+                    var waveFormat = wav.ToSampleProvider().WaveFormat;
```

Comment:
Alternative, conversion can be done like how non-wav files are handled. See https://github.com/stakira/OpenUtau/blob/05076e487213dca32660c378e83299b22de17182/OpenUtau.Core/Classic/VoicebankFiles.cs#L83-L93

Reply:
This is an error report and should be reported down to the smallest detail.
In fact, the current worldline will not play correctly if the sample rate is incorrect. There is also a possibility that voicebanks created by OU could be distributed and used by OG UTAU.